tables/de-g0-core.uti

```diff
@@ -615,6 +622,134 @@ comp6 \x00FD    6-2456      &#253;  LATIN SMALL LETTER Y WITH ACUTE
 comp6 \x00FE    6-1234      &#254;  LATIN SMALL LETTER THORN
 comp6 \x00FF   6-13456      &#255;  LATIN SMALL LETTER Y WITH DIAERESIS
 
+# --- Context rules ---
+
+# ensure begcapsword in places not handled automatically by Liblouis.
+# Extraneous capsletter signs removed in pass 2.
+noback context !$U[]$U!$l @45
+noback context !$U[]$U~ @45
+# The following lines currently makes liblouis loop over the first char (see #717)
```

Comment:
#717 has been fixed so the "workaround" is probably not needed anymore.

We should get rid of these `context` rules (and the corresponding `pass2` rules below) altogether though. Either these are needed because of a bug or shortcoming in `begcapsword`, or that was the case but Liblouis has been fixed in the meantime. What happens to the tests if you try disabling the rules?

Reply:
We didn't get rid of all the context rules (it's not possible, the reason is explained in a comment) but we got rid of the workaround.